macros/plugins/snowflake/refresh_external_table.sql

```diff
@@ -5,8 +5,9 @@
     
     {% set auto_refresh = external.get('auto_refresh', false) %}
     {% set partitions = external.get('partitions', none) %}
+    {% set delta_format = (external.table_format | lower == "delta") %}
     
-    {% set manual_refresh = (partitions and not auto_refresh) %}
+    {% set manual_refresh = not auto_refresh %}
```

Comment:
@danielefrigo explain why `partitions` is removed here

Reply:
With this condition, only partitioned tables are refreshed by the package.
There are multiple situations (delta tables is one of them) in which you want to manually refresh an external tables (e.g. you cannot setup Snowflake automatic refresh for technical reasons on the cloud provider), so I'd prefer to link this feature only to the `auto_refresh` flag